backend/wmg/data/cube_pipeline.py

```diff
@@ -41,13 +42,18 @@ def load(dataset_directory: List, corpus_path: str, validate: bool = False):
             gc.collect()
 
         logger.info("all loaded, now consolidating.")
+
         for arr_name in [f"{corpus_path}/{name}" for name in ["obs", "var", INTEGRATED_ARRAY_NAME]]:
             tiledb.consolidate(arr_name)
             tiledb.vacuum(arr_name)
 
 
 def load_data_and_create_cube(
-    path_to_h5ad_datasets: str, corpus_name: str = "corpus_group", snapshot_path=None, extract_data=True
+    path_to_h5ad_datasets: str,
+    corpus_name: str = "corpus_group",
+    snapshot_path=None,
+    extract_data=True,
+    validate_cubes=True,
```

Comment:
nit: If this method is never being called with `validate_cubes=False`, do we need the new param?

Reply:
This is based on your suggestion for making debugging validation failures easier to debug. This will skip the validation checks and copy to s3 without updating the snapshot id

Reply:
Because that isnt super straightforward, I'm going to create a ticket to do that in a separate pr (in the interest of getting this merged in asap).

Reply:
https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/2704